Feature/issue#10 Implement LIC6 method and tests

RasmussBrodin opened the pull request:
This commit addresses issue #10 by implementing the required LIC6 functionality. Currently, the structure differs from the previous pull request. Please share your thoughts on the following:

Should all LIC methods be placed in the same class/file, or should they be in Main? (SOLVED: place LIC methods in EvaluateLIC)
Should the LIC test cases be consolidated into a single test file, or should each LIC have its own test file? (SOLVED: all test cases relating to LIC will be placed in EvaluateLICTest)

RasmussBrodin committed:
Feat: Added class for LIC and method for LIC6

This commit will add a class for all LIC  methods and a method for LIC6 for later integration into the DECIDE method.

It will also create a new file containing tests for LIC6.

RasmussBrodin committed:
Feat: Small update for test testLIC6NeagtiveShort

Fixed incorrect parameter in test where numPoints should be less than 3.

RasmussBrodin committed:
Changed name of LIC test class

Changed the name of the LIC test class so all tests relating to LIC can be placed there.

Juliapp123 commented:
looks good, I think you should add tests for edge cases such as when numPoints = 3 and nPts = 3, and when all points lying on a single line. I also think that the testfile should have the same name as the file it is testing for with the ending Test. Finally, the comment explaining the function is not complete.

RasmussBrodin committed:
Fix: Added edge test cases and changed name of test file

Added test case for when all points are on the same line, and when nPts=numPoints=3.

Changed name of test file to EvaluateLICTest to conform with convention.

RasmussBrodin committed:
fix: Updated comment explaining function

Added missing text explaining the function of the method.

Juliapp123 commented:
looks good!
